examples/cfq/input_pipeline.py

```diff
@@ -97,7 +97,8 @@ def get_padded_shapes(self, output_len):
         constants.QUESTION_KEY: [None],
         constants.QUERY_KEY: [query_pad],
         constants.QUESTION_LEN_KEY: [],
-        constants.QUERY_LEN_KEY: []
+        constants.QUERY_LEN_KEY: [],
+        constants.QUERY_NO_BOS_LEN_KEY: []
```

Comment:
This seems redundant. Given that QUERY_NO_BOS_LEN_KEY is always QUERY_LEN_KEY-1, why do you have to store it separately? Can't you just compute one from the other when you need it?

Reply:
Most likely yes, but I will have to do some subtractions at batch level